sdk/eventhub/event-hubs/src/log.ts

```diff
@@ -20,3 +20,62 @@ export function logErrorStackTrace(error: unknown): void {
     logger.verbose(error.stack);
   }
 }
+
+/**
+ * Creates a logger that includes the connectionId, sender or receiver name, and
+ * client type.
+ * @internal
+ */
+export function createLogPrefix(
+  connectionId?: string,
+  type?: "Sender" | "Receiver" | "Management",
+  name?: string
+): string {
+  let prefix = "";
```

Comment:
style nit, prefer pushing the parts to an array and joining them:

```ts
const parts: string[] = [];
if (connectionId) {
  parts.push( `[${connectionId}]`);
}
if (type) {
  parts.push(type);
}
if (name) {
  parts.push(name);
}
return parts.join(" ");
```

Reply:
Good point, addressed in https://github.com/Azure/azure-sdk-for-js/pull/25214/commits/9e1faf1551c63799b5ad35689ebb081699067fa1.